Composer/packages/server/src/controllers/project.ts

```diff
@@ -50,7 +50,6 @@ async function createProject(req: Request, res: Response) {
     const currentProject = await BotProjectService.getProjectById(id, user);
     if (currentProject !== undefined) {
       await currentProject.updateBotInfo(name, description);
-      await currentProject.init();
```

Comment:
project.init() is already called on getProjectById() which is the single place of every project created. so here don't need call again,